D3D12Bundles/Shaders.cs

```diff
@@ -99,6 +106,17 @@ public class SimpleShader : IShader {
             return result;
         }
 
+        [ShaderMethod]
+        [Shader("vertex")]
+        public PSInput MutinyVSMain(MutinyVSInput input) {
```

Comment:
Alternate shader for use with Mutiny asset - the difference is `Mutiny` currently isn't providing normals or tangent data.